pkg/operation/seed/seed.go

```diff
@@ -393,6 +394,23 @@ func BootstrapCluster(ctx context.Context, k8sGardenClient, k8sSeedClient kubern
 			return err
 		}
 
+		if conf.Logging != nil && conf.Logging.Loki != nil && conf.Logging.Loki.GardenLoki != nil &&
+			conf.Logging.Loki.GardenLoki.PriorityClassValue != nil {
+			priority := *conf.Logging.Loki.GardenLoki.PriorityClassValue
+			if err := deletePriorityClassIfValueNotTheSame(ctx, k8sSeedClient.Client(), common.GardenLokiPriorityClassName, priority); err != nil {
+				return err
+			}
+			lokiValues["priorityClass"] = map[string]interface{}{
+				"value": priority,
+				"name":  common.GardenLokiPriorityClassName,
+			}
+		} else {
+			pc := &schedulingv1.PriorityClass{ObjectMeta: metav1.ObjectMeta{Name: common.GardenLokiPriorityClassName}}
+			if err := k8sSeedClient.Client().Delete(ctx, pc); client.IgnoreNotFound(err) != nil {
+				return err
+			}
```

Comment:
What about calling `deletePriorityClassIfValueNotTheSame` with `valueToCompare=-1`?

Reply:
What do you mean. I am not using uint. If the current value is not -1 the PC will be deleted.

Reply:
Yes, in this case you always want to delete the `PriorityClass`, and you already have a function for deleting it, so I suggest the following:

```suggestion
			if err := deletePriorityClassIfValueNotTheSame(ctx, k8sSeedClient.Client(), common.GardenLokiPriorityClassName, -1) != nil {
				return err
			}
```

The current value can never be `-1`, so it will always be deleted as desired.

Anyways, it doesn't make a big difference, so I'm approving the PR (thanks for addressing the other suggestions), this was just something that caught my eye :)